README.md

```diff
@@ -5,7 +5,12 @@
 ## Installation
 Using a python3 environment, run the following to install required libraries:
 ```
-pip install -r requirements.txt
+make build
```

Comment:
Since we don't have a contribution guide now, instead of replacing the old Installation instruction, do you mind adding another section:
```
## Installing through Docker
```

Reply:
Yes no problem

Reply:
How do I update the pull request

Reply:
Ok, I think I just updated the Readme file. Please let me know if there's anything I need to modify